interactions/api/models/guild.py

```diff
@@ -714,6 +753,283 @@ async def modify_member(
         )
         return Member(**res, _client=self._client)
 
+    async def get_preview(self) -> "GuildPreview":
+        """
+        Get the guild's preview.
+        """
+        return GuildPreview(**await self._client.get_guild_preview(guild_id=int(self.id)))
+
+    async def leave(self) -> None:
+        """
+        Removes the bot from the guild.
```

Comment:
I'm not sure if black formatted it (ignore if it did) but I believe the docstring should all be on one line for singular comments.

Reply:
Black ignores it in both cases